source/extensions/filters/http/transformation/inja_transformer.h

```diff
@@ -83,14 +84,31 @@ class Extractor : Logger::Loggable<Logger::Id::filter> {
                             const Http::RequestOrResponseHeaderMap &header_map,
                             GetBodyFunc &body) const;
 
+  // Matching enum with the protobuf definition
+  enum class Mode {
+    EXTRACT = 0,
+    SINGLE_REPLACE = 1,
+    REPLACE_ALL = 2
```

Comment:
Could we use the proto enum instead? It's one less thing to keep track of if the proto enum grows in the future

Reply:
definitely, good call

Reply:
Resolved here: https://github.com/solo-io/envoy-gloo/pull/301/commits/66c09331678aecc7a3cadce6094d472edefd4ee0